lib/matplotlib/_tff_header_parse.py

```diff
@@ -0,0 +1,401 @@
+
```

Comment:
It should be both py2 and py3 (but I work in 3 day-to-day).  Did I accidentally use a py3 only feature?